app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -0,0 +1,224 @@
+
+Call us on +44 800 389 2500 for:
+
+* guidance on the UK equivalents of your qualifications
+* a free statement of comparability, once you’ve submitted your application, if your provider asks for this
+* advice about what to do if you do not have English, maths and science qualifications of grade 4 GCSE standard
+
+
+### Find and apply for a teacher training course
+
+[Find postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses) lists courses leading to qualified teacher status by region, subject and age group, and guides you to application.
+ 
+### If you do not have a university degree
+
+If you do not have a degree, you can [apply to study a 3 to 4 year undergraduate course](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) in England which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+
+The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
+
+### Further information
+
+[Train to be a teacher](/train-to-be-a-teacher) gives more information about training in England. 
+
+[Funding your training](/funding-your-training) explains what financial support is available if you are in the UK under the [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa), the [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) or the [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme).
+
+## Tips for schools & training providers
+
+### Visas and immigration
+
+Most Ukrainian refugees will not need to be sponsored for a Skilled worker or Student visa. [Check the status of your visa](#check-the-status-of-your-visa) gives more information about visa schemes for Ukrainian refugees allowing them to live, work and study in the UK.
+
+### Qualifications
+
+Schools should assess the qualifications and experience of Ukrainian teachers in the same way as other overseas applicants, bearing in mind the exemption provided by the [4-year rule](https://www.gov.uk/guidance/recruit-teachers-from-overseas#employing-overseas-teachers-without-qts-the-4-year-rule).
+
+[UK ENIC](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) can provide a ‘statement of comparability’ to applicants showing how their academic and professional qualifications compare to the English equivalents. However, they will need scans or photographs of their original documents.
```

Comment:
I think we've already said this at least once, if not twice. Do we need to say it again? I'd be inclined to remove this para. But it you want to keep it, can we format it in the same way I've indicated for earlier instances please?